mono/metadata/marshal.h

```diff
@@ -772,11 +772,13 @@ ves_icall_System_ComObject_ReleaseInterfaces(MonoComObject* obj);
 gpointer
 ves_icall_System_ComObject_GetInterfaceInternal (MonoComObject* obj, MonoReflectionType* type, MonoBoolean throw_exception);
 
+TYPED_HANDLE_DECL (MonoComInteropProxy);
```

Comment:
I try to put the handle declarations near the struct definition try to to encourage folks to notice that it's a managed type with a handle and to write new code using handles, if possible.   In `object-internals.h`.

Reply:
I'll try that.